src/routes/Datastreams.js

```diff
@@ -56,7 +56,7 @@ let router = express.Router();
  * }
  **/
 
-router.get('/Datastreams', (req, res) => {
+router.get('/', (req, res) => {
```

Comment:
We are using snake_case for file names. I realize that you need camelCase naming to dynamically add endpoints in `sensorthings.js`, but we can probably convert the names from snake_case to camelCase at runtime. It's in the server boot up path, so it shouldn't affect performance.